soh/src/code/z_player_lib.c

```diff
@@ -572,6 +573,23 @@ uint8_t Player_IsCustomLinkModel() {
            (LINK_IS_CHILD && ResourceGetIsCustomByName(gLinkChildSkel));
 }
 
+// Alternate Equipment Loading function. (global function)
+// Checks if player can use new loading method, for Bottles Only.
+uint8_t Player_CanUseNewLoadingMethodBottle(Player* this) {
+    if (!CVarGetInteger("gAltLinkEquip", 1)) {
+        return false;
+    }
+
+    switch (this->leftHandType) {
+        case PLAYER_MODELTYPE_LH_BOTTLE:
+            if (ResourceGetIsCustomByName(gLinkBottleDL)) {
+                return true;
+            }
+            break;
+            return false;
+    }
```

Comment:
Unless you intended on handling more cases here, this could just be a single liner right?
```suggestion
    return this->leftHandType == PLAYER_MODELTYPE_LH_BOTTLE && ResourceGetIsCustomByName(gLinkBottleDL);
```